### DISCUSSION TITLE
Web-Version?

### DISCUSSION THREAD

**Discussion by zerbsama** [iSoron/uhabits#1623] [Feature Requests]
Hi, 
since all data is on the device only but it's backuped via Cloud, how about an optional 2nd mode to have it working with a web interface and store it therefore on the cloud..?
so it would be independend to the particular device...

**Comment by Tuistmessiah** [comment]
@iSoron I would be willing to work on this feature if you want it. I do web-dev but have no experience with mobile. Would it be interesting to discuss what API I could use and how I can contribute?

**Reply by v1carter** [reply]
@Tuistmessiah @iSoron  A web version would allow users to create and view habbits online
